### ISSUE TITLE
Editing Image Files Including Text?

### ISSUE THREAD

**Issue by xoderal** [VriskaSerket51/eastward.js#1]
Could I ask for your help in editing the text written in images? Please take a look at the examples I've attached. Is it possible to extract such assets using your program? I actually extracted all the assets once and checked all the *.PNG files, but I couldn’t find the one corresponding to the example in the results I extracted.
![스크린샷 2024-09-19 201620](https://github.com/user-attachments/assets/c3143ec5-71c9-400a-ae1c-c174e6d94bc2)
![스크린샷 2024-09-19 201730](https://github.com/user-attachments/assets/12e1be4a-0937-4b4d-a180-3e0ae55907e6)
![스크린샷 2024-09-19 201803](https://github.com/user-attachments/assets/c412f9a3-e1a9-48ab-afc3-e2a0f7849e1b)
![스크린샷 2024-09-19 202021](https://github.com/user-attachments/assets/c14979cb-981f-4717-bd29-c8c724264c31)

**Comment by VriskaSerket51** [comment]
Those files are `deck_pack` assets, which are not currently supported by eastward.js (but will be updated in some days).
Anyway, the first one, 'HELLO GROCERY' is in `deck/env/Sand/SandTownA.decks.psd`, which is actually in `gfx_main/8c73126dfa55463a08e828b1659a1a26`.
![decks.hmg](https://github.com/user-attachments/assets/781e9d25-cb54-4456-afa4-77eaca894c83)

I updated package to 0.0.48, which contains `hmg2png` and `png2hmg` options.

`eastward unzip --out ./ ./gfx_main.g`
`eastward hmg2png -r --in ./gfx_main --out ./gfx_main` (ignore errors)
...do file edit...
`eastward png2hmg -r --in ./gfx_main --out ./gfx_main` (also ignore errors)
`eastward zip --root ./gfx_main --out ./gfx_main.g`

**Comment by xoderal** [comment]
![20240920190728_1](https://github.com/user-attachments/assets/97bc3cda-9f91-4d84-8329-7ebd7ebcb9eb)

![스크린샷 2024-09-20 190913](https://github.com/user-attachments/assets/26946c71-0845-4728-a2f9-c25f51abcb86)


It worked!

When I run the game with the .g file generated from the last command, the main screen appears, but trying to start or load the game causes it to freeze with a black screen. There were also error messages when I executed the last command (screenshot included). However, after compressing it with a different program (ConsoleProgram_Eastward.exe), the game runs without issues, and the changes I made are reflected. Everything went smoothly up to the step 4 out of 5. It might be an issue with my computer's environment. 

Thank you. I've made significant progress. :)
